frontend/src/Editor/AppVersionsManager.jsx

```diff
@@ -241,6 +242,37 @@ const CreateVersionModal = function CreateVersionModal({
       createVersion(versionName, createAppVersionFrom);
     }
   };
+  const options = appVersions.map((version) => {
+    return { ...version, label: version.name, value: version.name };
+  });
+  const width = '100%';
+  const height = 32;
+  const darkMode = localStorage.getItem('darkMode');
```

Comment:
@manishkushare Here the `darkMode` is a string ('true').

Reply:
@arpitnath applied the suggested change, it is working fine now, thank you.